mantle/kola/harness.go

```diff
@@ -783,11 +783,28 @@ func runProvidedTests(testsBank map[string]*register.Test, patterns []string, mu
 			}
 		}
 
-		if caughtTestError {
+		allDenylisted := true
+		for name := range tests {
+			isDenylisted, err := testIsDenyListed(name)
+			if err != nil {
+				fmt.Printf("Error while checking denylist for test")
+				continue
+			}
+
+			if !isDenylisted {
+				allDenylisted = false
+				break
+			}
+		}
```

Comment:
Let's break this out into its own function so in the if statement below we can just call it `allTestsDenyListed(tests)` and we don't have this block of code interrupting the flow of reading the code.

Reply:
Sounds good! have the function in the new commit